REPL.cpp

```diff
@@ -97,17 +128,33 @@ bool REPL::ExecuteSwift(std::string line)
                                                                          "swift_repl_module",
                                                                          swift::PrimarySpecificPaths("", input.module_name),
                                                                          m_llvm_ctx));
-
     std::cout << "=========LLVM IR==========\n";
     std::string llvm_ir;
     llvm::raw_string_ostream str_stream(llvm_ir);
     llvm_module->print(str_stream, nullptr);
     str_stream.flush();
     std::cout << llvm_ir << std::endl;
+
     return true;
 #undef CHECK_ERROR
 }
 
+// ModifyAST performs four modifications on AST:
+//    - Add global variable of same type as last expression.
+//    - Modify last expression to be assignment to this global variable.
+//    - Wrap existing AST into a function called  __repl_x where x is the REPL line number
```

Comment:
Is it the xth line number or xth expression? What happens in the event of a multi-line expression?

Reply:
xth REPL expression. This doesn't really care about multiline expressions because it's at the AST level.